sklearn/sklearn-template/template/scripts/train.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Convenience script for running ML training jobs.
+#
+# Prerequisites:
+#   - Google Cloud SDK
+#
+# Globals:
+#   PROJECT_ID: Google Cloud project to use.
+#   BUCKET_ID: Google Cloud Storage bucket to store output.
+#
+# Arguments:
+#  $1: Path or BigQuery table to dataset for ML training and eval
```

Comment:
BQ table should be in the format of [PROJECT_ID.DATASET.TABEL_NAME]

Reply:
Added.